radsat-sk/operation/utility/RDebug.c

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file RDebug.c
+ * @date December 29, 2021
+ * @author Tyrel Kostyk
+ */
+
+#include <RDebug.h>
+#include <RUart.h>
+#include <stdio.h>
+#include <stdarg.h>
+
+
+/***************************************************************************************************
+                                            DEFINITIONS
+***************************************************************************************************/
+
+#define MAX_DEBUG_CHAR_LENGTH	(4096)
+
+
+/***************************************************************************************************
+                                             PUBLIC API
+***************************************************************************************************/
+
+void debugPrint(const char* stringFormat, ...) {
+#ifdef DEBUG

```

Comment:
Makes sense to wrap this function in a DEBUG configuration. Unfortunately, I changed the configuration status to "Release" with the dropdown menu by the hammer and your #ifdef statement on line 25 did not gray out.

Reply:
See issue #42 and the local fix we just described in #161